dist/tools/openocd/openocd.sh

```diff
@@ -201,7 +201,7 @@ do_debug() {
             -c 'gdb_port ${GDB_PORT}' \
             -c 'init' \
             -c 'targets' \
-            -c 'halt' \
+            -c 'reset halt' \
```

Comment:
I think this should be made configurable somehow so you can only set it for certain boards.

Reply:
good idea !

Reply:
@miri64 see #8856